src/components/Alert.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactElement } from 'react';
+import { ExclamationCircleIcon } from '@heroicons/react/24/solid'
+
+/**
+ * Creates an alert with the given string message.
+ * 
+ * @returns The alert component
+ */
+const Alert = (props: {message: string, className?: string}): ReactElement =>  {
+    return (
+        <div className={`flex items-start space-x-2 text-left bg-[#FFEDC0] rounded-lg p-3 text-sm ${props.className}`} role="alert">
+            <ExclamationCircleIcon className="w-12 fill-[#FEC12F] stroke-white"/>
+            <span className="font-medium">{props.message}</span>
```

Comment:
nit: `font-bold text-base` instead of `font-medium` to be more accurate to the figma design. I think border-radius would also be closer to `rounded-2xl` since it's 15px.

Reply:
Updated!